config/config.go

```diff
@@ -55,6 +57,7 @@ type SQLiteConfig struct {
 
 type SQSConfig struct {
 	Enabled          bool     `name:"enabled" default:"true" help:"Enable SQS protocol for queue" env:"ENABLED"`
+	Host             string   `name:"host" default:"localhost" help:"hostname or ip address for SQS protocol" env:"HOST"`
```

Comment:
Could we make the default `""` or `0.0.0.0`? This would be to preserve backwards-compatibility with anyone who decides to upgrade.

Reply:
Ok, removed the default.